src/lib/openapi/spec/group-user-model-schema.ts

```diff
@@ -6,14 +6,19 @@ export const groupUserModelSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['user'],
+    description: 'Details for the single user belonging to a group',
     properties: {
         joinedAt: {
+            description: 'A date when the user join the group',
             type: 'string',
             format: 'date-time',
+            example: '2023-06-30T11:41:00.123Z',
         },
         createdBy: {
+            description: 'A user who added the group user',
```

Comment:
Wow, that's a tricky thing to try to formulate. Maybe something like this? I'm not sure it's better or worse 😅 

```suggestion
            description: 'The username of the user who added this user to this group',
```